engine/src/main/java/org/terasology/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -81,6 +80,8 @@ public void initialise() {
 
         addDesiredStateChange(new BindFBO(SCENE_SHADOW_MAP));
         addDesiredStateChange(new SetViewportSizeOf(SCENE_SHADOW_MAP));
+        addDesiredStateChange(new Clear(true, true, false));
+        addDesiredStateChange(new DisableCapability(GL_CULL_FACE));
```

Comment:
I'd rename DisableCapability to simply Disable. But first let me see how you implemented this as right now I'm not sure if there should be a generic Disable state change or if there should be a DisableCullFace state change instead.